scepter.ts

```diff
@@ -59,7 +59,7 @@ interface Command {
 
 interface Event {
   trigger: string,
-  event (object: any): Promise<Message>  // TODO: is this correct? Verify with further examples
+  event (): Promise<void>  // TODO: is this correct? Verify with further examples
```

Comment:
Do we have any event implementation to confirm if this is correct?

Reply:
I'm not exactly sure how to get a function that takes any number of arguments of any type in typescript, but there are event handlers with multiple arguments (like `messageUpdate`).

I was able to get this working with a basic `console_log` module that logged sent messages to the console, but our modules don't seem to type-check yet, so this could be something to revisit when we get that to work.